Examples/Example/Snapshot Tests/BaseSnapshotTestCase.swift

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) 2018-present Frédéric Maquin <fred@ephread.com> and contributors.
 // Licensed under the terms of the MIT License.
 
+#if targetEnvironment(macCatalyst)
```

Comment:
Quick question about the snapshot tests and catalyst. Are they conditionally excluded because `iOSSnapshotTestCase` doesn't support catalyst? Did you run into issues?

Reply:
I could not use `iOSSnapshotTestCase` on macOS as following error.
<img width="826" alt="Screen Shot 2020-10-05 at 13 59 56" src="https://user-images.githubusercontent.com/154327/95041936-1f8c6600-0713-11eb-8778-72793ecc409a.png">
I tried `carthage update --platform macOS` but I got an error `Dependency "ios-snapshot-test-case" has no shared framework schemes for any of the platforms: Mac`.

I think `iOSSnapshotTestCase` does not support macOS.